tf-modules/archive/ingest-reporting.tf

```diff
@@ -1,5 +1,6 @@
 # Report executions
 resource "aws_sns_topic" "report_executions_topic" {
+  kms_master_key_id = "alias/aws/sns"
```

Comment:
Should this be an input variable to our tf modules? i.e. is this something we expect our users to want to customize or is the default key acceptable?

Reply:
Great question - I think that falls solidly into YAGNI territory. 

As this is entirely a compliance action and using AWS managed keys doesn't incur additional costs it seems the better default v.s. self-managed keys incur SM API lookup costs that scale.     While that's hardly prohibitive, adding the logic to either/or, while trivial, increases configuration complexity without being asked for and doesn't feel like an obvious/immediate need given nobody is doing it now. 

If we decide users want self-managed keys, that seems like a fine thing to punt to future-update land.